mender.go

```diff
@@ -448,7 +448,7 @@ func (m mender) GetRetryPollInterval() time.Duration {
 }
 
 func (m *mender) SetState(s State) {
-	log.Infof("Mender state: %v -> %v", m.state.Id(), s.Id())
+	log.Infof("Mender state: %s -> %s", stateNames[m.state.Id()], stateNames[s.Id()])
```

Comment:
I would do this bit differently.

Just add `MenderState` `String()` method that will satisfy `Stringer` interface:
```
func (ms MenderState) String() string {
	return stateNames[ms]
}
```

Here you can have then:
`log.Infof("Mender state: %s -> %s", m.state.Id(), s.Id())`